server/routes/api/snip.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express')
+const router = express.Router()
+const snipService = require('../../services/snip');
+const config = require('../../config');
+const lockTypes = require('../../constants/lockTypes');
+
+if (config.env == 'development')
+    router.get('/', async (req, res) => {
+        const snips = await snipService.getAllSnips();
+        res.json(snips);
+    })
+else {
+    router.get('/', (req, res) => {
+        res.send('Snappit v1.0');
+    })
+}
+
+/**
+ * @swagger
+ *  /api/snip/:url:
```

Comment:
Change `/api/snip/:url:` to `/api/snip/{url}:`. I think that's the way to document URL parameters.